### ISSUE TITLE
Check nomenclature, synonyms and hierarchy for neoplasm, cancer and carcinoma terms

### ISSUE THREAD

**Issue by paolaroncaglia** [EBISPOT/efo#353]
Background:
This came up while looking at some of the Zooma mappings for patients’ self-reported traits in the field of cancer codes. These were all assigned high-confidence scores by Zooma, but several are narrow mappings and so may be considered incorrect. A typical example of narrow mapping is that ‘lung cancer’ was mapped to ‘lung carcinoma’, where carcinomas are subtypes of cancer.
There were also some broad mappings, which are not conceptually incorrect; e.g., ‘skin cancer’ was mapped to ‘skin neoplasm’, where cancers are subtypes of neoplasms. Again, these were all assigned a high-confidence score by Zooma. 

Issues and proposed solutions:
(Some issues may depend on whether Zooma looks at labels and synonyms alike when mapping, or only at labels)
- [ ] First of all, check that a term's label corresponds to its description, i.e. if its label contains "cancer" or "carcinoma" etc, examine the term's definition to ensure it really describes a type of cancer, or carcinoma, etc. (Zoe's spreadsheet below contains definitions for that purpose.)
- [ ] Are synonyms correct in EFO? E.g. both ‘skin neoplasm’ and ‘skin carcinoma’ have the synonym ‘skin cancer’. ‘alternative_term’ in EFO2 will be carried over into EFO3 as ‘has_exact_synonym’, so we’d need to adjust the synonym specificity to broad or narrow as appropriate, especially if Zooma looks at synonyms; or remove non-exact synonyms.
- [ ] Are narrow mappings occurring because there is no broad term? If so create them.
- [ ] The hierarchy under ‘neoplasm’ is very flat. It may be slightly better in EFO3, and if looking at the inferred view, but some classification issues may not be solved by EFO3, plus if our users are only looking at the asserted view they will lose granularity. It’s likely that there’s room for improving the ontology hierarchy and axioms under ‘neoplasm’. We should also check if there’s any error. 

Overview of nomenclature:
TL;DR: carcinoma is_a cancer is_a neoplasm

The following details were extracted from NCI thesaurus, Merriam-Webster medical dictionary, Wikipedia and ICD-10.

Hierarchy according to NCIT:

neoplasm (synonyms: neoplasia, neoplastic disease, neoplastic growth, tumor/tumour(*))
---benign neoplasm (NOT cancer)
------papilloma
------cystadenoma
------lipoma
------others
---malignant neoplasm (synonyms: cancer, malignancy, malignant growth, malignant neoplastic disease, malignant tumor)
------carcinoma
------sarcoma
------lymphoma
------leukemia
------others
(*) ‘tumor’ tends to be used when a swelling or lump are observed, so not all neoplasms should be synonymous with tumors, e.g. blood ones.

Wikipedia  and ICD-10 divide neoplasms in benign, potentially malignant or “in situ” (e.g. carcinoma in situ), and malignant (ICD-10 has the additional grouping of uncertain/unknown). So they substantially agree with NCIT except for the borderline case of carcinoma (malignant according to NCIT, potentially malignant according to Wikipedia and ICD-10). We should align with NCIT as that’s the approved standard for cancer (and used by PDX).

**Comment by zoependlington** [comment]
We also have a problem of neoplasm terms being classified under the parent term of cancer, e.g. lymphoid neoplasm (EFO_0001642). These should be a child of neoplasm.

**Comment by paolaroncaglia** [comment]
@zoependlington thanks, those will be fixed!

**Comment by paolaroncaglia** [comment]
Note for self/EFO editor: another case that needs correcting is EFO:0007392 nervous system cancer, currently subclass of carcinoma.
Actually the whole hierarchy under carcinoma needs revising, there are several suspiciuos classes  there such as muscle cancer, spleen cancer...

**Comment by zoependlington** [comment]
I have pulled all descendants of cancer into one spreadsheet with synonyms and all parents to look through easier. It can be found here:
https://docs.google.com/spreadsheets/d/1hyFXddoxbeZ4pJhhy45OlzjsY2EpWVy8RgNXHl0eBHI/edit?usp=sharing

**Comment by zoependlington** [comment]
Received an email from Adam on 6/3/2019:

"Looking at EFO3 3.3.1, I can see that there is the "breast carcinoma"
term EFO_0000305, and a "breast cancer" term MONDO_0007254, and
"breast neoplasm" EFO_0003869. That's going to a bit confusing to our
users, but I can accept that it is technically the correct medical
classification.

However, EFO_0000305 ("breast carcinoma") has synonyms including
"Breast Neoplasm" and "Breast Cancer" - which are the labels of the
other terms.

It would be ideal if we could have labels / synonyms that do not
overlap, if such a thing is possible? Or at least overlap less?
Otherwise it is going to be quite confusing for our search results,
and for data mapping e.g. with Zooma/OLS."

I directed him to this ticket. Simon suggested any parts that are causing problems, Adam should let us know so we can work on it. @paolaroncaglia Should we create a ticket to sort breast cancer/neoplasm/carcinoma?

**Comment by paolaroncaglia** [comment]
@zoependlington 
Sure, we can create a ticket to sort the breast-related terms, but I suspect that there will be quite a few cases that will cause confusion for Open Targets search results and mappings... e.g. a quick look at the spreadsheet https://docs.google.com/spreadsheets/d/1hyFXddoxbeZ4pJhhy45OlzjsY2EpWVy8RgNXHl0eBHI/edit#gid=870881862 with the labels sorted in alphabetical orders shows that 'bladder carcinoma' has synonyms for cancer and neoplasm, and that is only at line 62 of 485... we could probably perform an ad-hoc query to identify problematic cases, but I suspect that it would be a medium-effort task at least. So yes let's fix the breast cancer terms and then discuss priority of fixing all other 'overlaps'.

**Comment by paolaroncaglia** [comment]
At the OntoTools scrum today we resolved that:
- [ ] Plan is to tackle terms on a case by case basis as they are flagged - Paola opening a ticket re: Breast cancer (will add link here)
- [ ] After the re-import/mapping to NCIT it may be worth looking at this in more detail

**Comment by zoependlington** [comment]
Natalie has pointed out a few cases where the hierarchy is wrong.
For example: https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0001642
lymphoid neoplasm has the child myeloid neoplasm, which is very incorrect.

**Comment by NatConte** [comment]
file created in 2017 and ticket to EFO back then
https://www.ebi.ac.uk/panda/jira/browse/FGPTO-1298

[review ontology letter.docx](https://github.com/EBISPOT/efo/files/2985770/review.ontology.letter.docx)

**Comment by paolaroncaglia** [comment]
When it comes to classification of neoplasms, we may also want to keep in mind the following suggestions from @gkos-bio, stemming from a discussion on ICD10 mappings and how to classify subtypes of secondary malignant neoplasm in EFO:

"...that’s the complicated part with ICD10 and Cancer.
My understanding is that ‘Secondary malignant neoplasm’ is considered in the differential diagnosis of lesions in patients with a history of a specific cancer or used for billing or for assessing recurrence of cancer.
The shortcut is to map to Neoplasm as you suggested, but this is not good enough.
I don’t have a straight answer.
I suppose it depends how much modelling work you want to put in defining primary and secondary and metastatic stages in Cancer.
I’m afraid it’s not a trivial exercise.
And if you look at the hierarchy under
https://www.icd10data.com/ICD10CM/Codes/C00-D49/C76-C80/C78-
https://www.icd10data.com/ICD10CM/Codes/C00-D49/C76-C80/C79-
it means that EFO needs to cover all the organs or sites.
Unfortunately, we are not covering all the subtypes from ICD10 and this might cause problems in EFO in the future.
So You may want to model ‘neoplastic status’ as a property as in NCIt to make this compatible.
You may want to model both ‘primary’ and ‘secondary’ too when defining a class of subclass. So could be by is_a or by a property.
Alternatively you may want to add all the secondary malignant neoplasm subtypes labels as synonyms to metastatic malignant neoplasm in specific site.
e.g. Metastatic Malignant Neoplasm in the Small Intestine: https://www.ebi.ac.uk/ols/ontologies/ncit/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNCIT_C7524
Alternatively, you may want to create a new secondary malignant neoplasm class in EFO, cross-referencing to NCIt and classify anything from ICD10 under it.
https://www.ebi.ac.uk/ols/ontologies/ncit/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNCIT_C4968
Or a combination of all the approaches that will be consistent with the anatomical classification for all diseases in EFO/MONDO outside Cancer."

**Comment by paolaroncaglia** [comment]
Another level of discussion pertains to anatomical classification of neoplasm types. @gkos-bio also noted: 
"I had a look at MONDO for breast neoplasm and this was classified as a child of thorax neoplasm.
I’m wondering if this is 100% correct. In case of metastasis, it’s frequent to see migration to thoracic tissues but the ‘is_a’ might be misleading.
It looks as if the classification mimics the UBERON classification which can be frequently incorrect when it comes to disease classification."
I replied: 
"This comment, next to MONDO 'secondary malignant neoplasm', may help: "Note that we currently treat secondary neoplasms as being neoplastic diseases that are derived from neoplasm; classes such as 'neoplasm', 'carcinoma' are implicitly primary. This may change in future." So MONDO breast neoplasm is meant as primary neoplasm that arises in breast, and the UBERON classification would hold, but I agree it's something to keep in mind."

**Comment by zoependlington** [comment]
Plan is to look into importing NCIT through mapping eventually and reconfigure the cancer branch as a whole.

We can fix any examples that are listed above or if any users request a cancer term to be fixed in the mean time.

**Comment by zoependlington** [comment]
In an email with Maria Paula Magarinos, it has been highlighted that there is confusion for mapping as to which terms to choose for Open Targets, for example:

MONDO:0007576 and EFO:0002916,
MONDO:0005411 and EFO:1001956,
MONDO:0001056 and EFO:0000178,
MONDO:0008170 and EFO:0001075

**Comment by paolaroncaglia** [comment]
A note that at our meeting with Open Targets today, @d0choa and @smnorthen mentioned that, after fixing the Orphanet-derived parents issue (hopefully by their February release), the "next big thing" for them would probably be the revision of the neoplasm/cancer branch in EFO.

**Comment by paolaroncaglia** [comment]
A note that Mondo has a whole node under 'cell proliferation disorder', including neoplasms, so hopefully we could leverage that too (along with NCIT if necessary) to improve the representation of corresponding terms in EFO (we may want to confirm with Mondo first if that's an advisable strategy).

**Comment by paolaroncaglia** [comment]
Tagging this for potential proposal. May mostly ignore the long thread of comments and focus on the following:
- [ ] Zoe will pull out EFO-only children of 'neoplasm';
- [ ] A review of those would be most helpful to ensure correct placement.

**Comment by paolaroncaglia** [comment]
Note for self/EFO editors: 
When the neoplasm/cancer classification issues are fixed, check that the 'cancer cell line' branch is in good shape too, see https://github.com/EBISPOT/efo/issues/739#issuecomment-607236356.

**Comment by zoependlington** [comment]
Note to self: This should all be fixed now, but probably needs checking briefly.
